client/src/dojo/createOptimisticSystemCalls.ts

```diff
@@ -284,6 +286,30 @@ export function createOptimisticSystemCalls({
     };
   }
 
+  //   TODO: implement optimistic spawn npc
+  function optimisticSpawnNpc(systemCall: (args: SpawnNpcProps) => Promise<void>) {
```

Comment:
is this still todo or is it implemented?

Reply:
I don't think we will spawn NPCs from the frontend anymore so I can remove

Reply:
This will have to be removed later when we start touching the cairo contracts, meanwhile I'll leave it there